scripts/WinInstallFiles/LaunchJMRI.nsi

```diff
@@ -743,9 +747,18 @@ Section "Main"
 
   ; -- Launch the Java class.
   LaunchJMRI:
-  DetailPrint "Launching JMRI"
-  ; -- use $7 to hold return value
-  ExecWait `"$JEXEPATH" $OPTIONS -Djava.class.path="$CLASSPATH" $CLASS $PARAMETERS` $7
+    DetailPrint "Launching JMRI"
+    ; -- use $7 to hold return value
+    StrCmp $NOISY ${SW_NORMAL} IsNoisyExec
+    ExecWait `"$JEXEPATH" $OPTIONS -Djava.class.path="$CLASSPATH" $CLASS $PARAMETERS` $7
+    Goto IsNotNoisy
```

Comment:
Is $7 used only for ExecWait or also for StrCmp? If it's only used for ExecWait, it seems to me that the comment on line 751 should be after the StrCmp on line 752. But I don't fully understand this code.

Maybe the comment on line 751 should also be added before line 757?

Reply:
done